react-components/src/components/Header.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import '../styles/Header.css';
+
+export function Header(): JSX.Element {
+  return (

```

Comment:
Does not show current route.

Reply:
Please give me more information about this mistake.

Reply:
_Add header which shows the current page_
_Routing is enabled, header shows current route._

It should list either path or show text which route is active (Main, etc.)

Reply:
In any case, I can’t fix anything, so if you don’t agree with the score, then put the one you think is necessary.
(Q&A I was missed)